src/mappings/entities/mapMultiSig.ts

```diff
@@ -158,6 +159,21 @@ const handleMultiSigProposalAdded = async (
   const multisigId = getTextValue(rawMultiSigAddress);
   const proposalId = getNumberValue(rawProposalId);
   const creatorAccount = extrinsic?.extrinsic.signer.toString();
+  const parsed = (extrinsic?.extrinsic?.toHuman() as any).method.args || [];
+
+  const module = parsed[1]?.value.call_module;
+  const call = parsed[1]?.value.call_function;
+  const callIndex = parsed[1]?.value.call_index;
+  const expiry = parsed[2]?.value;
+  const args = parsed[1]?.value.call_args;
+
+  const proposalParams: MultiSigProposalParams = {
+    module,
+    call,
+    callIndex,
+    expiry,
+    args,
+  };
```

Comment:
In case of bridge transaction, these won't be there . Check - https://polymesh-testnet.subscan.io/block/12136029?tab=event